docs/cloud/provision/README.md

```diff
@@ -510,6 +510,8 @@ For our example, here is the complete form:
 
 ::: note
 Multiple peer routes are useful in case you want to peer with a subnet that has multiple IP ranges (aliases). A typical example would be aif you have a VPC-native GKE cluster, and you need pods in that cluster to work with Event Store Cloud. Then, you need to add the pod IP range to the peer route in addition to the subnet's primary IP range.
+
+After a peering is created, the peer routes cannot be changed. If there is a need to modify them, the peering should be deleted and created again with the updated set of peer routes. This is true even in the cases, where a cloud provider supports dynamic route updates, e.g. in GCP. The reason is that the EventStore automation creates firewall rules for the peer routes at the moment the peering is created. The firewall rules are not updated dynamically to prevent security breaches.
```

Comment:
```suggestion
:::

::: warning
After you create a peering, you can't change the peer routes. If you would need to modify the routes, delete the peering and create it again with the updated set of peer routes. The limitation applies even in cases where a cloud provider supports dynamic route updates, e.g. in GCP. The reason is that the EventStore automation creates firewall rules for the peer routes when the peering is created. The firewall rules are not updated dynamically to prevent security breaches.
```

Reply:
For me, the text is very clear. It's the same for all the clouds, whether the provider can dynamically update the routes or not.